docs-content/upgrading.html.md.erb

```diff
@@ -0,0 +1,95 @@
+---
+title: Updating CyberArk Conjur Service Broker for PCF
+owner: Partners
+---
+
+This topic explains how to upgrade the CyberArk Conjur Service Broker for Pivotal Cloud Foundry (PCF) tile.
+
+## <a id="upgrade"></a> Upgrade CyberArk Conjur Service Broker for PCF
+
+Upgrading the CyberArk Conjur Service Broker for PCF tile generally requires the same steps in
+the Ops Manager as when you first install the tile. Your configuration settings will migrate to
+the new version when you add the upgrade tile to the Ops Manager.
+
```

Comment:
Upgrading the CyberArk Conjur Service Broker for the PCF tile requires the same steps  you ran in the Ops Manager when you first installed the tile.  Your configuration settings migrate to the new version when you add the upgrade tile to the Ops Manager.